WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2047,6 +2047,16 @@ public void onAddMediaClicked() {
     }
 
     @Override
+    public void onTextDropped(String text) {
+        mEditorFragment.insertContent(text);
```

Comment:
It's not necessary per se. I've added it so to match the `onMediaDropped` being defined in the same interface.

More than happy to demote it and directly implement it inside the `EditorFragment.java`.